content/en/docs/languages/net/instrumentation.md

```diff
@@ -7,6 +7,20 @@ description: Instrumentation for OpenTelemetry .NET
 
 {{% docs/languages/manual-intro %}}
 
+{{% alert title="Note" color="info" %}}
+
+On this page you will learn how you can add traces, metrics and logs to your
+code manually. You are not limited to using one kind of instrumentation:
+you can also use [automatic instrumentation](/docs/languages/net/automatic/)
+to get started and then enrich your code with manual instrumentation as needed.
+
+Also, for libraries your code depends on, you don't have to write
+instrumentation code yourself, since they might come with OpenTelemetry
```

Comment:
Minor, I find "come with OpenTelemetry" a bit confusing, consider:

"since they might be already instrumented or there are instrumentation libraries for them"